nilearn/plotting/img_plotting.py

```diff
@@ -1252,6 +1281,9 @@ def plot_stat_map(
         Default='continuous'.
     %(radiological)s
 
+    kwargs: extra keyword arguments, optional
+        Extra keyword arguments passed to TODOODO
```

Comment:
If you do this, we'll be able to merge ;-)

Reply:
Actually I am having second thoughts about some of this PR...

Let me ping you at the relevant spot so you can tell me what you think.